examples/todo/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "counter",
```

```suggestion
  "name": "valtio-todo-example",
```

ooops, just one more.